install.rst

```diff
@@ -0,0 +1,114 @@
+Install an OMERO server
+=======================
+
+This chapter describes an installation of an OMERO.server and the most important post-installation steps, such as additional apps installation and LDAP configuration. The installation of the server on local machine is not recommended, except for testing or development purposes. The workflow is aimed at system administrators intending to familiarize themselves with OMERO installation.
+
+Description
+-----------
+
+We will show here:
+
+
+-  Where to find the OMERO.server requirements.
+
+-  How to install OMERO.server quickly using a simple Ansible playbook example.
+
+-  How to install OMERO.server very quickly using a provided Docker image.
+
+-  How to configure OMERO.server to work with LDAP, both using Ansible or using manual configuration.
+
+-  How to install applications for OMERO.web, such as OMERO.figure.
+
+Resources
+---------
+
+- Webpage  `Start with OMERO at your Institution <https://www.openmicroscopy.org/omero/institution/getting-started.html>`_
+- `Ansible <https://www.ansible.com/>`_
+- `Ansible documentation <https://docs.ansible.com/ansible_community.html>`_
+- `Example <https://github.com/ome/omero-deployment-examples>`_ and `production <https://github.com/ome/prod-playbooks>`_ ansible playbooks.
+- `OMERO installation workshop presentation <https://downloads.openmicroscopy.org/presentations/2020/Dundee/Workshops/OME2020-OMERO-Installation/#/>`_
+- `Manual installation documentation <https://omero.readthedocs.io/en/v5.6.5/sysadmins/unix/server-centos7-ice36.html>`_.
+- `System requirements <https://omero.readthedocs.io/en/v5.6.5/sysadmins/system-requirements.html>`_
+- `List of Apps for OMERO.web and Command line plugins <https://www.openmicroscopy.org/omero/apps/>`_
+
+Install a server
+----------------
+
+Get an empty box or virtual machine with CentOS 7.
```

Comment:
Still ``empty box``
Maybe indicate that the installation steps described below have been tested on CentOS 7 only.
Someone may want to install on Ubuntu20.04 for example

Reply:
Fixed it also here in https://github.com/ome/omero-guides/pull/236/commits/1f7e120d4562c583678acd6d3d61f2c0f26d1dab - I also accepted all the other changes in the ``install.rst`` - maybe I can close the other PR and continue working just here on one PR only